### PR TITLE
Fix explanation of logging configuration

### PR DESCRIPTION
**Pull request by mudassir014**
Corrected some issues at https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#configure-logging to correctly bridge the content and the preceding JSON.

Took reference from https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#configure-logging

[Internal Preview URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0&branch=pr-en-us-24595#configure-logging).

### CODE REVIEW

**Comment by mudassir014** [comment]
The similar section at https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#configure-logging has some additional information as well. Let me know if I should add that as well.

**Comment by serpent5** [comment]
Hi @mudassir014. Thanks for taking the time to open a PR.

The section you've changed describes the *appsettings.Development.json* file generated by the templates, which includes the `Microsoft.AspNetCore` property. As the section is intended to be an explanation of the template-generated configuration, I think we should leave the explanation as-is.

**Comment by serpent5** [comment]
Having said that, I think the current explanation suggests that the `Microsoft` category exists in the preceding JSON, but it doesn't. @Rick-Anderson Should we do something about that?

File: aspnetcore/fundamentals/logging/index/samples/6.x/TodoApiDTO/appsettings.Development.json
Comment (serpent5) [inline aspnetcore/fundamentals/logging/index/samples/6.x/TodoApiDTO/appsettings.Development.json:5]:
```suggestion
      "Microsoft.AspNetCore": "Warning"
```

File: aspnetcore/fundamentals/logging/index.md
Comment (serpent5) [inline aspnetcore/fundamentals/logging/index.md:76]:
```suggestion
* The `"Microsoft.AspNetCore"` category applies to all categories that start with `"Microsoft.AspNetCore"`. For example, this setting applies to the `"Microsoft.AspNetCore.Routing.EndpointMiddleware"` category.
```

File: aspnetcore/fundamentals/logging/index.md
Comment (serpent5) [inline aspnetcore/fundamentals/logging/index.md:77]:
```suggestion
* The `"Microsoft.AspNetCore"` category logs at log level `Warning` and higher.
```

File: aspnetcore/fundamentals/logging/index.md
Comment (serpent5) [inline aspnetcore/fundamentals/logging/index.md:75]:
```suggestion
* The `"Default"` and `"Microsoft.AspNetCore"` categories are specified.
```

**Comment by serpent5** [comment]
Thanks again @mudassir014. I think this latest version looks good, but I'll wait for @Rick-Anderson to review it before considering a merge.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@mudassir014 really nice fix. Keep them coming. 
Be sure to check out your name as a contributor about the 3rd of next month on the [What's new page](https://docs.microsoft.com/en-us/aspnet/core/whats-new/) in the **Community contributors** section at the bottom.

**Comment by mudassir014** [comment]
@Rick-Anderson Will do.
@Rick-Anderson @serpent5 Thank you for everything. :)

**Pull request merged**
